analyses/molecular-subtyping-ATRT/01-ATRT_subtyping.R

```diff
@@ -58,10 +69,11 @@ methyl_map <- atrt_df_methyl %>%
   mutate(molecular_subtype_methyl = molecular_subtype)
 
 # any dups? no
-length(unique(methyl_map$id)) == length(methyl_map$id)
+length(unique(methyl_map$match_id)) == length(methyl_map$match_id)
 
 
-# for the samples, whose cns_methlation_subclass_score >= 0.8 and dkfz_v12_methylation_subclass is one of the three types in ATRT_subtype_list, their molecular subtype are same as dkfz_v12_methylation_subclass
+# for the samples, whose dkfz_v12_methylation_subclass_score >= 0.8 and dkfz_v12_methylation_subclass is one of the three types in ATRT_subtype_list, 
+# their molecular subtypes are same as dkfz_v12_methylation_subclass
```

Comment:
```suggestion
# for the samples, whose dkfz_v12_methylation_subclass_score >= 0.8 and dkfz_v12_methylation_subclass is one of the three types in ATRT_subtype_list,
# or 
# NIH_v2_methylation_Class_mean_score >= 0.9 & NIH_v2_methylation_Class_mean_score >= 0.9
# their molecular subtypes are same as dkfz_v12_methylation_subclass/NIH_v2_methylation_Class
```

Reply:
So, I wrote the code in a way to be preferential to the DKFZ subclass, so only if DKFZ is <0.8, it would take NIH